src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -454,25 +484,48 @@ private AssemblyResult getAssemblyResult(final Haplotype refHaplotype, final int
 
         printDebugGraphTransform(rtgraph, refHaplotype.getLocation() + "-sequenceGraph." + kmerSize + ".0.1.cleaned_readthreading_graph.dot");
 
-        final SeqGraph initialSeqGraph = rtgraph.toSequenceGraph();
-        if (debugGraphTransformations) {
-            initialSeqGraph.printGraph(new File(debugGraphOutputPath, refHaplotype.getLocation() + "-sequenceGraph." + kmerSize + ".0.1.initial_seqgraph.dot"), 10000);
-        }
+        // Either return an assembly result with a sequence graph or with an unchanged sequence graph deptending on the kmer duplication behavior
+        if (generateSeqGraph) {
```

Comment:
If it's possible for `generateSeqGraph` to be `false` inside a method called `cleanUpSeqGraph` then either I misunderstand something (totally possible, of course!) or something needs to be renamed.

Reply:
I don't think this is an issue. This line is in the getAssemblyResult class which may either contain a seq graph or a read threading graph depending on the mode.